apps/payments/next/app/[locale]/layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { app } from '@fxa/payments/ui/server';
```

Comment:
```suggestion
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import { app } from '@fxa/payments/ui/server';
```

I should look at creating an ESLint rule for this so the linter catches missing copyrights automatically; I know I used to have a custom rule that did that, and I think [Monitor](https://github.com/mozilla/blurts-server/blob/b3993582610430238773444fb848aa7ca81db7ed/.eslintrc.json#L24-L30) found some plugin that will do it (at the expense of another dependency). 🤔